StreamVideo.xcodeproj/project.pbxproj

```diff
@@ -4774,6 +4774,7 @@
 				DEVELOPMENT_ASSET_PATHS = "\"DemoApp/Preview Content\"";
 				DEVELOPMENT_TEAM = EHV7XZLAHA;
 				ENABLE_PREVIEWS = YES;
+				GCC_OPTIMIZATION_LEVEL = s;
```

Comment:
which targets have this setting and why?

Reply:
Demo app in `test` config, to speed up the build process. `Release` config has this by default, `debug` does not because it includes some debug stuff in the build and builds longer.